pkg/converter/convert.go

```diff
@@ -217,6 +217,32 @@ func Convert(o Options, pathOptions *clientcmd.PathOptions) error {
 				exec.Args = append(exec.Args, argIsLegacy)
 			}
 
+		case token.WebBrowserLogin:
+
+			if argClientIDVal == "" {
+				return fmt.Errorf("%s is required", argClientID)
+			}
+
+			exec.Args = append(exec.Args, argClientID)
```

Comment:
you are missing server ID

Reply:
The code already added server ID for all the mode as shown in the image below. 
![image](https://user-images.githubusercontent.com/80654129/192194537-ee25edaf-59bd-4a8c-92d1-91034582c873.png)